cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -394,6 +396,13 @@ func (s *Server) Run() {
 		switch state {
 		case indexStateSuccess:
 			log.Printf("updated index %s in %v", args.String(), elapsed)
+			logger.Info("updated index",
+				zap.String("repo", args.Name),
+				zap.Uint32("id", args.RepoID),
+				zap.Strings("branches", args.BranchesStrings()),
+				zap.Int64("duration_ms", elapsed.Milliseconds()),
+				zap.String("duration_str", elapsed.String()),
```

Comment:
similarly use zap.Duration
```suggestion
				zap.Duration("duration", elapsed),
```

Reply:
Updated, I'll add a comment to the updated file showing the formatted log output